packages/feedback/src/environments/environment.prod.ts

```diff
@@ -0,0 +1,11 @@
+export const environment = {
+  production: true,
+  name: 'production',
+  api: 'https://api.one.redhat.com',
+  graphqlAPI: 'https://api.one.redhat.com/graphql',
+  client: 'https://one.redhat.com',
+  authIssuer: 'https://auth.redhat.com/auth',
+  matomoSiteID: 10, // Site ID from Piwik Analytics
+  serviceNowURL: 'https://redhat.service-now.com',
+  subscription: 'wss://api.one.redhat.com/subscriptions'
+};
```

Comment:
Can you please remove this hosts from here and please use the .env files for managing this information